packages/react-server-webpack-plugin/src/test/fixtures/no-entrypoints/webpack.config.js

```diff
@@ -1,33 +1,39 @@
+const path = require('path');
 const {ReactServerPlugin} = require('../../../react-server-webpack-plugin');
 
-const universal = {
-  mode: 'production',
-  optimization: {
-    minimize: false,
-  },
-  plugins: [new ReactServerPlugin()],
-};
-
-const server = {
-  ...universal,
-  name: 'server',
-  target: 'node',
-  entry: './server',
-  externals: [
-    (context, request, callback) => {
-      if (/node_modules/.test(context)) {
-        return callback(null, `commonjs ${request}`);
-      }
-      callback();
+module.exports = function config(usesNode) {
+  const universal = {
+    mode: 'production',
+    optimization: {
+      minimize: false,
     },
-  ],
-};
+    plugins: [new ReactServerPlugin()],
+    resolve: {
+      modules: ['node_modules', path.resolve(__dirname)],
```

Comment:
Will `path.resolve` be `app/ui` for Rails tests?

Reply:
I don't think it will, and I think it's a smell that the test setup is too oblique.  Consider just creating `test/fixtures/rails` and `test/fixtures/node` and duplicating setups.

Reply:
> Will path.resolve be app/ui for Rails tests?

Not necessarily. The need for `app/ui` is based on the assumption that a project is using sewing-kit rails setup, which these tests aren't.

This config adds the root path to the module resolution so `import App from 'index';` resovles to `__dirname/index.js`